docs/source/es/_toctree.yml

```diff
@@ -28,10 +28,14 @@
   - local: custom_models
     title: Compartir modelos personalizados
   - sections:
+    - local: tasks/question_answering
+      title: Contestación de preguntas
```

Comment:
Maybe Respuesta a preguntas is a better translation

Reply:
That's what was used at https://nlp-en-es.org/recursos/tutoriales/01_aplicaciones_de_los_transformers